tests/legacy/unit-tests/templates/functions.php

```diff
@@ -178,4 +178,16 @@ function( $gateways ) {
 
 		$this->assertEquals( $expected_html, $actual_html );
 	}
+
+	public function test_hidden_field() {
+		$actual_html = woocommerce_form_field('test',
+		array(
+			'type' => 'hidden',
+			'id' => 'test_field',
+		), 'test value');
+		$expected_html = '<input type="hidden" class="input-hidden" name="test" id="test_field"  value="test value" />';
```

Comment:
This function will actually output something more like: `<p class="form-row " id="test_field_field" data-priority=""><span class="woocommerce-input-wrapper"><input type="hidden" class="input-hidden " name="test" id="test_field"  value="test value"  /></span></p>` since it wraps it in a container.